sdk/appconfiguration/tests.yml

```diff
@@ -5,4 +5,5 @@ jobs:
     parameters:
       ServiceDirectory: appconfiguration
       EnvVars:
+        AZURE_TEST_MODE: RECORD
         AZCONFIG_CONNECTION_STRING: $(java-azconfig-test-connection-string)
```

Comment:
Tests are failing because it's expecting this to be `AZURE_APPCONFIG_CONNECTION_STRING` instead of `AZCONFIG_CONNECTION_STRING`

Reply:
Changed